### PR TITLE
Add Search Icon to Config Dashboard

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

Adds icon that uses Quick Bar

Adds header to Quick bar to allow closing

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by zsarnett** [comment]
Gif
image
image

File: src/translations/en.json [hunk 1]
```diff
@@ -626,7 +626,9 @@
             "server_control": "[%key:ui::panel::config::server_control::caption%]"
           }
         },
-        "filter_placeholder": "Entity Filter"
+        "filter_placeholder": "Entity Filter",
+        "title": "Quick Search",
+        "key_c_hint": "Did you know you can click c on any page to open this search bar?"
```
Comment (spacegaier) [inline src/translations/en.json:631]:
```suggestion
        "key_c_hint": "Did you know you can press "c" on any page to open this search bar?"
```

**Review by spacegaier — DISMISSED** [review]

File: src/translations/en.json [hunk 2]
```diff
@@ -626,7 +626,9 @@
             "server_control": "[%key:ui::panel::config::server_control::caption%]"
           }
         },
-        "filter_placeholder": "Entity Filter"
+        "filter_placeholder": "Entity Filter",
+        "title": "Quick Search",
+        "key_c_hint": "Did you know you can press "c" on any page to open this search bar?"
```
Comment (zsarnett) [inline src/translations/en.json:631]:
```suggestion
        "key_c_hint": "Did you know you can press 'c' on any page to open this search bar?"
```

File: src/translations/en.json [hunk 3]
```diff
@@ -626,7 +626,9 @@
             "server_control": "[%key:ui::panel::config::server_control::caption%]"
           }
         },
-        "filter_placeholder": "Entity Filter"
+        "filter_placeholder": "Entity Filter",
+        "title": "Quick Search",
+        "key_c_hint": "Did you know you can press 'c' on any page to open this search bar?"
```
Comment (balloob) [inline src/translations/en.json:631]:
```suggestion
        "key_c_hint": "Press 'c' on any page to open this search bar"
```

**Review by balloob — DISMISSED** [review]

**Review by balloob — DISMISSED** [review]

**Review by balloob — APPROVED** [review]

**Pull request merged**